skeleton/webpack/gulpFile.js

```diff
@@ -0,0 +1,10 @@
+const del = require('del');
+const gulp = require('gulp');
+const path = require('path');
+const project = require('./aurelia_project/aurelia.json');
+
+function clearDist() {
+  return del([path.resolve(__dirname, project.platform.output)]);
```

Comment:
We can remove gulp and del from webpack app :-)
https://github.com/johnagan/clean-webpack-plugin

Reply:
There might be a downside to it. I have not yet tested it but my hypothesis is that it might create a conflict with the build steps that create compiled output in the `dist` folder as a pre-webpack build step, and then during webpack build, this plugin cleans the `dist` folder. An example of such build step can be to generate the locale files in a gulp task. I think the use cases can be explained in README, and the alternatives, `del` and `clean-webpack-plugin` are presented to devs.

Reply:
You are the webpack guy :), so the decision is yours.
Our flex design holds us back, if there is no `project.platform.output`, it would be just `"del scripts && ..."` in npm scripts.

Reply:
_Please_ remove gulp and del if possible. Can't you use [npm scripts](https://docs.npmjs.com/misc/scripts)?

Reply:
Exactly. With `npm scripts` it will be lot easier, only thing is that the output directory path comes from the config file. The path can be replicated and hardcoded in `package.json`, but there is a good chance that will create maintenance overhead. I also don't like the gulp file. I think I will go with the webpack plugin and add some info about that in README.